<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Crossfader</class>
 <widget class="QWidget" name="GUI_Crossfader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>44</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="grid_layout">
   <property name="horizontalSpacing">
    <number>20</number>
   </property>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cb_active">
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QLabel" name="lab_crossfader">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="SayonaraSlider" name="sli_crossfader">
     <property name="minimum">
      <number>2000</number>
     </property>
     <property name="maximum">
      <number>15000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1000</number>
     </property>
     <property name="value">
      <number>5000</number>
     </property>
     <property name="sliderPosition">
      <number>5000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SayonaraSlider</class>
   <extends>QSlider</extends>
   <header>GUI/Helper/SayonaraWidget/SayonaraSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
